src/CodingMilitia.PlayBall.Shared.StartupTasks/DbInitializer.cs

```diff
@@ -0,0 +1,31 @@
+﻿using AspNetCore.AsyncInitialization;
+using CodingMilitia.PlayBall.GroupManagement.Data;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using System.Threading.Tasks;
+
+namespace CodingMilitia.PlayBall.Shared.StartupTasks
+{
+    public class DbInitializer : IAsyncInitializer
+    {
+        private IWebHost _host;
+        public DbInitializer(IWebHost host)
+        {
+            this._host = host;
+        }
+
+        public async Task InitializeAsync()
+        {
+            using (var scope = _host.Services.CreateScope())
+            {
+                var hostingEnvironment = scope.ServiceProvider.GetRequiredService<IHostingEnvironment>();
+                if (hostingEnvironment.IsDevelopment() || hostingEnvironment.IsEnvironment("DockerDevelopment"))
+                {
+                    var context = scope.ServiceProvider.GetService<GroupManagementDbContext>();
```

Comment:
Instead of referencing `GroupManagementDbContext` directly, we can change `DbInitializer` to `DbInitializer<T>` , where `T` is a `DbContext`.
This way we can reuse this initializer for multiple databases 🙂

Reply:
![image](https://user-images.githubusercontent.com/15646312/53098994-cf718780-355f-11e9-8b91-27cb358cae39.png)
I am having this error. What might be the problem? I already have the dependency with EFCore

Reply:
Nevermind this error, resolved this issue by adding PostgreSQL package

Reply:
Yep.
I was checking it, by looking [here](https://docs.microsoft.com/en-us/dotnet/api/microsoft.entityframeworkcore.relationaldatabasefacadeextensions.migrateasync?view=efcore-2.1).
Says it's in assembly `Microsoft.EntityFrameworkCore.Relational`, looked for a NuGet for this and it exists. The PostgreSQL package has a reference to it, so it works as well 👍